packages/devtools_app/lib/src/screens/performance/tabbed_performance_view.dart

```diff
@@ -82,8 +87,20 @@ class _TabbedPerformanceViewState extends State<TabbedPerformanceView>
       ),
     );
 
+    final rebuildMetrics = RebuildMetricsView(
```

Comment:
super nit: should we rename to rebuildStats and RebuildStatsView to have consistent naming with feature flag and other names?

Reply:
done. removed all uses of the word metric[s].